elastic/kibana
# [Stack Monitoring] health api support for data ingested from package

### Summary

Closes #138864

Add support for data ingested from packages in the `metrics-{product}.*` patterns.

This change only fetches the data for the successfully monitored products, but we can also retrieve the errors like we do for metricbeat since an `error.message` may be populated on failure. This will be handled in https://github.com/elastic/integrations/issues/4011

### Testing
- setup a stack with an agent - you can follow the testing steps for the logstash package in https://github.com/elastic/integrations/pull/3993 to start an elastic-package stack with elasticsearch and logstash data
- connect your local kibana to the elastic-package elasticsearch - [howto](https://github.com/elastic/observability-dev/blob/548dfc0c5fe2776af40c1e85adc515c38139b4e5/docs/infra-obs-ui/stack-monitoring_integration-packages.md#connecting-a-local-kibana)
- hit `localhost:5602/api/monitoring/v1/_health` - a monitored cluster should show up with cluster, elasticsearch and logstash data collected with the `package` method
